tests/test_app/tests/managementcommand_tests.py

```diff
@@ -86,7 +87,7 @@ def ls_test(*args, expected):
             lines = out.read().splitlines()
             self.assertEqual(sorted(lines), sorted(expected))
 
-        test_data = (
+        test_data: T.List[T.Tuple[T.List[str], T.List[str]]] = [
```

Comment:
This isn't strictly necessary for the Pyright tests, but it's a more sensible data structure and made MyPy happier.